source/developers/concepts/hooks_extension_framework.rst

```diff
@@ -0,0 +1,155 @@
+=========================
+Hooks Extension Framework
+=========================
```

Comment:
```suggestion
#########################
```

In docs.openedx.org, we use the following headings:

H1: `#` (4 lines in the character)
H2: `*` (3 lines)
H3: `=` (2 lines)
H4: `-`

Reply:
Thank you! Done: https://github.com/openedx/docs.openedx.org/pull/599/commits/a88af4d9f0b50091f95ef1ac2a07c7d1d7bd1872